src/main/java/duke/task/Task.java

```diff
@@ -0,0 +1,66 @@
+package duke.task;
+
+/**
+ * Represents a task.
+ */
+public class Task {
+
+    protected int id;
+    protected String description;
+    protected boolean isDone;
+
+    /**
+     * Constructor for task.
+     * @param description Name of the command.
+     * @param id ID of task
+     */
+    public Task(String description, int id) {
+        this.description = description;
+        this.id = id;
+        this.isDone = false;
+    }
+
+    /**
+     * Marks task as done.
+     */
+    public void markDone() {
+        isDone = true;
+    }
+
+    /**
+     * Gets the ID of the task.
+     * @return The ID of the task.
+     */
+    public int getID() {
+        return id;
+    }
+
+    /**
+     * Gets the description of the task.
+     * @return The description of the task.
+     */
+    public String getDescription() {
+        return description;
+    }
+
+    /**
+     * Checks if the task is complete.
+     * @return A boolean value representing the completion status of the task.
+     */
+    public boolean isDone() {
```

Comment:
This method seems like a getter used to fetch the value of isDone variable. Perhaps you could consider changing this to getIsDone() so as to avoid confusions between the class variable and class method.